content/en/docs/performance/resource_consumption.md

```diff
@@ -0,0 +1,188 @@
+---
+draft: false
+linktitle: Kmesh Resource Consumption
+menu:
+  docs:
+    parent: performance
+    weight: 2
+title: Kmesh Resource Consumption
+toc: true
+type: docs
+---
+# 1. Background Information
+eBPF programs consume a certain amount of CPU and memory resources during their execution, which may affect the performance of other applications. To gain a deeper understanding of Kmesh’s resource consumption under different loads, we conducted several CPU and memory stress tests. These tests aim to evaluate the limits of CPU and memory consumption of Kmesh in real-world usage scenarios.
```

Comment:
This is a small amount, i would suggest removing `which may effect the performance of other applications`